Sourcery/Templates/Description.stencil

```diff
@@ -1,9 +1,9 @@
-{% for type in types.classes %} {% if type.variables.count %}{% if not type.annotations.skipDescription %}
+{% for type in types.classes|!annotated:"skipDescription" %} {% if type.variables.count %}
 extension {{ type.name }} {
     override var description: String {
         var string = {% if type.inheritedTypes.first == "NSObject" %}"\(type(of: self)): "{% else %}super.description{% endif %}
-        {% for variable in type.variables|!annotated:"skipDescription" %}string += "{{ variable.name }} = \({{ variable.name }}){% if not forloop.last %}, {% endif %}"
+        {% for variable in type.variables|!annotated:"skipDescription" %}string += "{{ variable.name }} = \(self.{{ variable.name }}){% if not forloop.last %}, {% endif %}"
```

Comment:
Need `self.` here because we now have ``` `throws` ``` variable but back ticks are not in the name. Might be a bug in variable parsing or something we can improve. Though using `self` is a valid workaround I think.